code/modules/antagonists/abductor/equipment/abduction_gear.dm

```diff
@@ -518,7 +518,7 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 		toggle_on(user)
 
 	if(iscyborg(target))
-		if(BATON_STUN)
```

Comment:
lol how do u even find these

Reply:
Its due a new feature that is referenced in one of these other PR's or one with updating spacemandmm or something else I think it was.